app/models/order_import.rb

```diff
@@ -110,18 +110,24 @@ def handle_save_clean_orders # TODO refactor and rename
 
   def handle_save_nothing_on_error # TODO refactor and rename
     Order.transaction do
-      CSV.open(upload_file_path, headers: true, skip_lines: /^,*$/).each do |row|
-        row_importer = import_row(row)
-        self.error_report += row_importer.row_with_errors.to_csv
-
-        if row_importer.errors?
-          result.failures += 1
-        else
-          result.successes += 1
+      begin
+        CSV.open(upload_file_path, headers: true, skip_lines: /^,*$/).each do |row|
+          row_importer = import_row(row)
+          self.error_report += row_importer.row_with_errors.to_csv
+
+          if row_importer.errors?
+            result.failures += 1
+          else
+            result.successes += 1
+          end
         end
-      end
 
-      raise ActiveRecord::Rollback if result.failed?
+        raise ActiveRecord::Rollback if result.failed?
+      rescue ArgumentError => e
```

Comment:
Do we want to rescue more than just ArgumentError? I could see us wanting to catch everything (except Rollback, but that could possibly be taken out of the begin block).